webapp/components/RelativeDateTime/index.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <span>{{ relativeDateTime }}</span>
+</template>
+
+<script>
+import formatRelative from 'date-fns/formatRelative'
+import { enUS, de, nl, fr, pt, es /*, pl*/ } from 'date-fns/locale'
+const locales = {
+  en: enUS,
+  de,
+  nl,
+  fr,
+  es,
+  pt
+  //pl
```

Comment:
What's wrong with Polish? :thinking: 

@ulfgebhardt by the way, I think I missed one test case: What happens, if you have a `i18n.locale()` but this locale is not in the list above? Could you create a test case for it? I would like to have a default in that case.

Reply:
Polish is currently not implemented on library side.